core/src/main/java/com/linecorp/armeria/common/DefaultHttpHeaders.java

```diff
@@ -174,6 +175,32 @@ public HttpHeaders status(HttpStatus status) {
         return status(status.code());
     }
 
+    @Override
+    public MediaType contentType() {
+        final MediaType contentType = this.contentType;
+        if (contentType != null) {
+            return contentType;
+        }
+
+        final String contentTypeString = get(HttpHeaderNames.CONTENT_TYPE);
+        if (contentTypeString == null) {
+            return null;
+        }
+
+        try {
+            return MediaType.parse(contentTypeString);
```

Comment:
Cache the media type for multiple calls?

this.contentType = parse
return this.contentType